MN/Algoritmi/Gauss/G/G.m

```diff
@@ -0,0 +1,12 @@
+function [A, b] = G(A, b)
+	n = size(A)(1);
+	for p = 1 : n - 1
+		for i = p + 1 : n	
+			factor = A(i, p) / A(p, p);
+			% calcularea vectorizata a matricei sistemului si a
+			% coloanei termenilor liberi									
+			A(i, :) = A(i, :) - factor * A(p, :);
+			b(i, :) = b(i, :) - factor * b(p, :);
+		endfor
+	endfor
+endfunction

```

Comment:
inainte de endfunction ar trebui apelat un SST cum fac eu la Householder.

Reply:
pot apela SST daca este in alt folder sau trebuie copiat?